components/brave_rewards/browser/promotion.h

```diff
@@ -21,6 +21,7 @@ struct Promotion {
   uint64_t expires_at;
   uint32_t type;
   uint32_t status;
+  bool legacyClaimed;
```

Comment:
There is also an iOS 'Promotion' object, should that be updated with this property as part of this PR?